### PR TITLE
ZeroClipboard 2.x.x

### PR DESCRIPTION
**Pull request by leonyu**
The existing ZeroClipboard is covers mostly `v1.x.x`. 

I have done the following: 
- move existing file to `zeroclipboard-1.x.x.d.ts`
- Make typing for `2.x.x`
- Tests using code example for `2.x.x` from 
  - from https://github.com/zeroclipboard/zeroclipboard/blob/master/docs/api/ZeroClipboard.md
- Tests using code example for `1.x.x` from 
  - from https://github.com/zeroclipboard/zeroclipboard/blob/1.x-master/docs/instructions.md

### CODE REVIEW

**Comment by leonyu** [comment]
In regards to the checklist, `zeroclipboard/zeroclipboard-1.x.x.d.ts` is a straight move of the existing file which didn't have tests.

**Comment by leonyu** [comment]
I have since added test for version `1.x.x`. It is however hard to add docs for it, since the documentation is dated.

**Comment by leonyu** [comment]
@ejsmith @niemyjski @balassy If you guys are still using ZeroClipboard in your project, could you take look to see if my changes are correct? Thank you in advanced.

File: zeroclipboard/zeroclipboard.d.ts [hunk 1]
```diff
@@ -1,75 +1,505 @@
-// Type definitions for ZeroClipboard
-// Project: https://github.com/jonrohan/ZeroClipboard
-// Definitions by: Eric J. Smith <https://github.com/ejsmith>, Blake Niemyjski <https://github.com/niemyjski>, György Balássy <https://github.com/balassy>
```
Comment (vvakame) [inline zeroclipboard/zeroclipboard.d.ts]: please keep original author names. add your name below.

**Comment by vvakame** [comment]
I'll merge this pull request. please ping me again.

File: zeroclipboard/zeroclipboard.d.ts (hunk 1 above)
Comment (leonyu) [inline zeroclipboard/zeroclipboard.d.ts]: Reinstated

**Comment by leonyu** [comment]
@vvakame done

**Comment by vvakame** [comment]
thanks mate!

**Pull request merged**